### PR TITLE
Add individual docs for rotate_{left, right}

### PR DESCRIPTION
**Pull request by llogiq**
r? @QuietMisdreavus

### CODE REVIEW

File: src/libcore/num/mod.rs [hunk 1]
```diff
@@ -2138,57 +2140,58 @@ assert_eq!(n.trailing_zeros(), 3);", $EndFeature, "
...
+integer.
+
+Please note this isn't the same operation as `>>`!
+
+# Examples
+
+Basic usage:
+
+Please note that this example is shared between integer types.
+Which explains why `u64` is used here.
```
Comment (QuietMisdreavus) [inline src/libcore/num/mod.rs]: Should this note be taken out?

**Comment by QuietMisdreavus** [comment]
```
tidy check
[00:03:47] tidy error: /checkout/src/libcore/num/mod.rs:2048: line longer than 100 chars
[00:03:49] some tidy checks failed
```

File: src/libcore/num/mod.rs (hunk 1 above)
Comment (llogiq) [reply src/libcore/num/mod.rs]: Yes. I'll do it.

**Comment by llogiq** [comment]
Off-by-one errors are still the bane of programming...

**Comment by QuietMisdreavus** [comment]
@bors r+

Thanks!

**Pull request merged**